### PR TITLE
Add new loss with prox computed by `svmbir.recon`

### LINKED ISSUE #2: support use of svmbir prox mode as an x step solver in ADMM
SVMBIR provides a method for the proximal map associated with f(x) = || Ax - y ||^2_W (https://svmbir.readthedocs.io/en/latest/theory.html). We want to provide a way to use it inside ADMM.

### PR DESCRIPTION
**Pull request by Michael-T-McCann**
Includes the code and tests, but currently no example.

Closes #2.

### CODE REVIEW

File: scico/linop/radon_svmbir.py [hunk 1]
```diff
@@ -88,3 +89,49 @@ def _bproj_hcb(self, y):
             y,
             result_shape=jax.ShapeDtypeStruct(self.input_shape, self.input_dtype),
         )
+
+
+class SvmbirWeightedSquaredL2Loss(WeightedSquaredL2Loss):
```
Comment (bwohlberg) [inline scico/linop/radon_svmbir.py:94]: Perhaps `SVMBIRWeightedSquaredL2Loss`?

File: scico/linop/radon_svmbir.py [hunk 2]
```diff
@@ -88,3 +89,49 @@ def _bproj_hcb(self, y):
...
+            num_rows=self.A.input_shape[1],
+            num_cols=self.A.input_shape[2],
+            sigma_p=np.float(sigma_p),
+            sigma_y=1.0,
+            positivity=False,
+            verbose=0,
+        )
+        return result
+        # return jax.experimental.host_callback.call(
+        #     lambda x: svmbir.recon(
```
Comment (bwohlberg) [inline scico/linop/radon_svmbir.py:131]: Worth keeping this commented-out code in the module?
